setting_up_firely_server/configuration/appsettings.rst

```diff
@@ -95,6 +95,43 @@ JSON formally has no notion of comments. But the configuration system of ASP.Net
 
 This will ignore the AutoUpdateConnectionString.
 
+.. _configure_settings_path:
+
+Providing settings in a different folder
+----------------------------------------
+
+It can be useful or even necessary to provide settings outside of the Firely Server folder itself, e.g. when mounting the settings to a Docker container. That is possible. 
+
+1. Provide an environment variable named ``VONK_PATH_TO_SETTINGS``, set to the folder where the settings are to be read from. This path can be absolute or relative to the Firely Server directory.
+2. In this folder you must provide at least one of the following files:
+
+   1. ``appsettings.instance.json``
+   2. ``logsettings.instance.json``
+   3. ``auditlogsettings.instance.json``
+
+3. These files will be read with the same :ref:`priority <configure_levels>` as they would have if they were in the Firely Server directory. 
+
+Note that if you provide this environment variable, then:
+
+#. The designated folder must exist.
+#. At least one of the three files must be present.
+#. The account that runs Firely Server must have read access to each of the files.
```

Comment:
```suggestion
#. The account that runs Firely Server must have read access to at least one of the files.
```

Reply:
If the file exists, the ConfigurationBuilder will try to read it. And if any of them is then not readable, it will still result in an error, albeit not by our code. So the documentation is still correct I think.
It is very unlikely that there are >1 files in the settings folder, with 1 of them not being readable, so let's keep the code as it is for now (I just merged it...)